client/test/unit/node/index.ts

```diff
@@ -0,0 +1,46 @@
+import glob from "glob";
```

Comment:
Why was this introduced?

Reply:
test:unit script use this file to get every unit test file and perform them.
npm run test:unit will execute all unit tests 
npm run test:unit {filename} will execute all unit tests for that file